### PR TITLE
1.13.0 updated

### PR DESCRIPTION
**Pull request by Subhashvysyaraju**
## Description

Updated sdk version 1.13.0

### CODE REVIEW

File: SDK/AppCenterAnalytics/Microsoft.AppCenter.Analytics.UWP/Microsoft.AppCenter.Analytics.UWP.csproj [hunk 1]
```diff
@@ -11,11 +11,12 @@
     <AssemblyName>Microsoft.AppCenter.Analytics</AssemblyName>
     <DefaultLanguage>en-US</DefaultLanguage>
     <TargetPlatformIdentifier>UAP</TargetPlatformIdentifier>
-    <TargetPlatformVersion>10.0.14393.0</TargetPlatformVersion>
+    <TargetPlatformVersion>10.0.17763.0</TargetPlatformVersion>
```
Comment (guperrot) [inline SDK/AppCenterAnalytics/Microsoft.AppCenter.Analytics.UWP/Microsoft.AppCenter.Analytics.UWP.csproj]: Please revert changes in this file, this is not part of the version bump script and should not be touched during a release.

File: SDK/AppCenter/Microsoft.AppCenter.UWP/Microsoft.AppCenter.UWP.csproj [hunk 2]
```diff
@@ -11,11 +11,12 @@
     <AssemblyName>Microsoft.AppCenter</AssemblyName>
     <DefaultLanguage>en-US</DefaultLanguage>
     <TargetPlatformIdentifier>UAP</TargetPlatformIdentifier>
-    <TargetPlatformVersion>10.0.15063.0</TargetPlatformVersion>
+    <TargetPlatformVersion>10.0.17763.0</TargetPlatformVersion>
```
Comment (guperrot) [inline SDK/AppCenter/Microsoft.AppCenter.UWP/Microsoft.AppCenter.UWP.csproj]: Please revert this file, this is not part of the version bump script and should not be touched during a release.

File: SDK/AppCenter/Microsoft.AppCenter.UWP/project.json [hunk 3]
```diff
@@ -1,18 +0,0 @@
-﻿{
```
Comment (guperrot) [inline SDK/AppCenter/Microsoft.AppCenter.UWP/project.json]: restore this file

File: SDK/AppCenterAnalytics/Microsoft.AppCenter.Analytics.UWP/project.json [hunk 4]
```diff
@@ -1,16 +0,0 @@
-﻿{
```
Comment (guperrot) [inline SDK/AppCenterAnalytics/Microsoft.AppCenter.Analytics.UWP/project.json]: restore this file

File: SDK/AppCenterPush/Microsoft.AppCenter.Push.UWP/Microsoft.AppCenter.Push.UWP.csproj [hunk 5]
```diff
@@ -11,12 +11,13 @@
     <AssemblyName>Microsoft.AppCenter.Push</AssemblyName>
     <DefaultLanguage>en-US</DefaultLanguage>
     <TargetPlatformIdentifier>UAP</TargetPlatformIdentifier>
-    <TargetPlatformVersion>10.0.14393.0</TargetPlatformVersion>
+    <TargetPlatformVersion>10.0.17763.0</TargetPlatformVersion>
```
Comment (guperrot) [inline SDK/AppCenterPush/Microsoft.AppCenter.Push.UWP/Microsoft.AppCenter.Push.UWP.csproj]: Please revert this file, this is not part of the version bump script and should not be touched during a release.

File: SDK/AppCenterPush/Microsoft.AppCenter.Push.UWP/project.json [hunk 6]
```diff
@@ -1,17 +0,0 @@
-﻿{
```
Comment (guperrot) [inline SDK/AppCenterPush/Microsoft.AppCenter.Push.UWP/project.json]: restore this file

File: Tests/Microsoft.AppCenter.Test.UWP/Microsoft.AppCenter.Test.UWP.csproj [hunk 7]
```diff
@@ -11,13 +11,14 @@
     <AssemblyName>Microsoft.AppCenter.Test.UWP</AssemblyName>
     <DefaultLanguage>en-US</DefaultLanguage>
     <TargetPlatformIdentifier>UAP</TargetPlatformIdentifier>
-    <TargetPlatformVersion>10.0.15063.0</TargetPlatformVersion>
+    <TargetPlatformVersion>10.0.17763.0</TargetPlatformVersion>
```
Comment (guperrot) [inline Tests/Microsoft.AppCenter.Test.UWP/Microsoft.AppCenter.Test.UWP.csproj]: Please revert this file, this is not part of the version bump script and should not be touched during a release.

File: Tests/Microsoft.AppCenter.Test.UWP/project.json [hunk 8]
```diff
@@ -1,16 +0,0 @@
-﻿{
-  "dependencies": {
-    "Microsoft.NETCore.UniversalWindowsPlatform": "5.2.6",
```
Comment (guperrot) [inline Tests/Microsoft.AppCenter.Test.UWP/project.json]: restore this file

File: SDK/AppCenterAnalytics/Microsoft.AppCenter.Analytics.UWP/Microsoft.AppCenter.Analytics.UWP.csproj (hunk 1 above)
Comment (Subhashvysyaraju) [reply SDK/AppCenterAnalytics/Microsoft.AppCenter.Analytics.UWP/Microsoft.AppCenter.Analytics.UWP.csproj]: reverted the changes

File: SDK/AppCenter/Microsoft.AppCenter.UWP/Microsoft.AppCenter.UWP.csproj (hunk 2 above)
Comment (Subhashvysyaraju) [reply SDK/AppCenter/Microsoft.AppCenter.UWP/Microsoft.AppCenter.UWP.csproj]: reverted the changes

**Pull request merged**